data/src/main/java/com/bottlerocketstudios/mapsdemo/data/network/YelpService.kt

```diff
@@ -10,5 +10,7 @@ internal interface YelpService {
     suspend fun getBusinessesByLatLng(
         @Query("latitude") latitude: Double,
         @Query("longitude") longitude: Double,
+        @Query("limit") limit: Int,
+        @Query("radius") radius: Int?,
```

Comment:
@Josh-Gebbeken It would be nice to have some inline comments mentioning snippets from the api docs. Since I just looked these up, I've added them in a suggestion here:

```suggestion
        // Optional. Number of business results to return. By default, it will return 20. Maximum is 50.
        @Query("limit") limit: Int,
        // Optional. A suggested search radius in meters. This field is used as a suggestion to the search.
        // The actual search radius may be lower than the suggested radius in dense urban areas, and higher in regions of less business density.
        // If the specified value is too large, a AREA_TOO_LARGE error may be returned. The max value is 40000 meters (about 25 miles).
        @Query("radius") radius: Int?, 
```

Reply:
@Josh-Gebbeken Additionally, it would be nice to refactor `radius` to `radiusMeters` for clarity